test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -36,7 +36,10 @@ func TestMeshGatewayDefault(t *testing.T) {
 		"global.federation.createFederationSecret": "true",
```

Comment:
I think this test is supposed to be disabled? I think I am missing where it's disabled and wasn't able to catch that.

Reply:
We're disabling tproxy for these tests below explicitly. Sorry for the inconsistency, I just thought it would still be valuable to run these in the nightly instead of skipping them like we do with the other test.